test/TestProjectWithSDKRef/TestProjectWithSDKRef.csproj

```diff
@@ -9,6 +9,6 @@
     </ItemGroup>
 
     <ItemGroup>
-        <PackageReference Include="TestProject" Version="#{NBGV_NuGetPackageVersion}#" />
+        <PackageReference Include="TestProject" Version="#{DependencyVersion}#" />
```

Comment:
What if we were to define two `PackageReference`'s here, one with `Condition="'$(UseFloatingVersions)'=='True'"` and one with `!=True` and then have two different tokens (one using a floating version and one without). Then we would only have to add `/p:UseFloatingVersions=True` to the `dotnet build` command depending on which SDK we're currently using?

Reply:
Are you thinking that `UseFloatingVersions` property would be just for the test project? If so, I think that could make sense. I'd probably tweak the name though, as both versions technically use floating versions, it's just a question of single vs. double wildcards.

Reply:
Yeah, sure, I think that should be something we'd use for testing only. I don't expect users to do it this way.